4-projects/modules/single_project/main.tf

```diff
@@ -33,6 +33,9 @@ module "project" {
   shared_vpc         = local.host_network.project
   shared_vpc_subnets = local.host_network.subnetworks_self_links # Optional: To enable subnetting, to replace to "module.networking_project.subnetwork_self_link"
 
+  vpc_service_control_attach_enabled = var.vpc_service_control_attach_enabled
+  vpc_service_control_perimeter_name = var.vpc_service_control_perimeter_name
+
   labels = {
```

Comment:
would not be better to just receive a `labels` object here?

Reply:
yeah I think a `map` makes more sense in terms of flexibility to allow arbitrary labels but I will probably refrain from interface changes in this PR. We can pick this up after.